install_files/ansible-base/roles/validate/tasks/validate_gpg_info.yml

```diff
@@ -1,20 +1,24 @@
 ---
 
-- name: Validate GPG info.
+- name: Validate GPG fingerprints.
   assert:
     that:
```

Comment:
Found a great edge case for you :) I typed the file name wrong and it passed this section but when it went to the script it failed and it was driving me crazy. Just says the fingerprints dont match (so i assumed it was reading my file in wrong). Can you detect if the file actually exists here?